### PR TITLE
chore: flame_behaviors-v0.2.0

### PR DESCRIPTION
**Pull request by renancaraujo**
## Description

chore: flame_behaviors-v0.2.0

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [x] 🗑️ Chore

### CODE REVIEW

File: packages/flame_behaviors/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
-# [0.1.1](https://github.com/VeryGoodOpenSource/flame_behaviors/compare/v0.1.0...v0.1.1) (2022-06-20)
+# [0.2.0](https://github.com/VeryGoodOpenSource/flame_behaviors/compare/v0.1.1...v) (2023-01-23)
+
+### Features
+
+* add dependabot ([#31](https://github.com/VeryGoodOpenSource/flame_behaviors/issues/31)) ([4b601a1](https://github.com/VeryGoodOpenSource/flame_behaviors/commit/4b601a1ff3a516e36ae850857f5c5e5a9de7303f))
+* update version constraints ([#26](https://github.com/VeryGoodOpenSource/flame_behaviors/issues/26)) ([05303be](https://github.com/VeryGoodOpenSource/flame_behaviors/commit/05303beeae80df6055f3b3fc8f5630f2643d40fe))
```
Comment (wolfenrain) [inline packages/flame_behaviors/CHANGELOG.md:6]: #34 is missng 👀

Comment (renancaraujo) [reply packages/flame_behaviors/CHANGELOG.md:6]: we have a problem, conventional changelog cli doesnt recognize these commits

Comment (wolfenrain) [reply packages/flame_behaviors/CHANGELOG.md:6]: Time to do things manually :D

Comment (renancaraujo) [reply packages/flame_behaviors/CHANGELOG.md:6]: it is going to be really fun in the future

Comment (wolfenrain) [reply packages/flame_behaviors/CHANGELOG.md:6]: This is why I use [a script](https://github.com/wolfenrain/sashimi/blob/main/tools/release_ready.sh)

**Review by wolfenrain — APPROVED** [review]

**Review by erickzanardo — APPROVED** [review]

**Pull request merged**